app/views/posts/_list.html.erb

```diff
@@ -1,4 +1,5 @@
 <% posts.each do |post| %>
+  <% if post.is_drafted == false %>
```

Comment:
Can you refactor this ?

Reply:
No magesh it is already written there are just fields in there iam rendering drafts in another partial(_draftlist.html.erb).so the initial condition is necessary to differenciate the partials